smoke/go_test.go

```diff
@@ -0,0 +1,77 @@
+package smoke_test
```

Comment:
Are we actually using this test? It seems like `init_test.go` only invokes the nodejs test.

Reply:
You are correctly I removed all of the tests other than the Node.js one since the builder initially will only have Node.js support.

I'm a bit surprised go did not complain about it, since it complains about most un-used things. Will remove in next commit